addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -229,8 +234,15 @@ class DescriptionScreen extends Component {
      * @param {Object} ui an object with label and value properties for
      *      the selected option.
      */
-    _selectIndustry(ev, ui) {
+    async _selectIndustry(ev, ui) {
         if (this.autocompleteHasResults) {
+            if (this.sendIap.length) {
+                await this.orm.call('website', 'configurator_send_iap', [], {
+                    'attempt': this.sendIap,
+                    'final_industry_choice': ui.item.label,
```

Comment:
You can probably omit the `final_` part, the shorter the better, but here it doesn't bring anything anyway as there is only one industry choice at the end: the final one